packages/lab/src/dropdown-next/DropdownNext.tsx

```diff
@@ -0,0 +1,256 @@
+import { clsx } from "clsx";
+import { ListNext, ListNextProps } from "../list-next";
+import {
+  makePrefixer,
+  useId,
+  useForkRef,
+  UseFloatingUIProps,
+  SaltProvider,
+} from "@salt-ds/core";
+import { ChevronDownIcon, ChevronUpIcon } from "@salt-ds/icons";
+import {
+  useRef,
+  forwardRef,
+  HTMLAttributes,
+  FocusEvent,
+  KeyboardEvent,
+  MouseEvent,
+  MutableRefObject,
+} from "react";
+import { useWindow } from "@salt-ds/window";
+import dropdownNextCss from "./DropdownNext.css";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { FloatingPortal, Placement } from "@floating-ui/react";
+import { useDropdownNext } from "./useDropdownNext";
+
+const withBaseName = makePrefixer("saltDropdownNext");
+
+export interface DropdownNextProps
+  extends Pick<UseFloatingUIProps, "open" | "onOpenChange" | "placement">,
+    HTMLAttributes<HTMLElement> {
+  /**
+   * If `true`, dropdown will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * Initially selected value for the dropdown.
+   */
+  defaultSelected?: string;
+  /**
+   * List of options when using a dropdown.
+   */
+  source: string[];
```

Comment:
Is drop down only for strings, or can it have numbers as well ?

Reply:
source takes only string arrays so numbers will need to be converted into strings